### PR TITLE
Mimic gRPC-java ToAllUperCase() method

### LINKED ISSUE #27: Wrong uppercasing of method names
The method we use:
```java
        public String methodNameUpperUnderscore() {
            return CaseFormat.LOWER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, methodName);
        }
```
doesn't match the one used by grpc-java.

Proto method `CreateEntityWithDTO` becomes `CREATE_ENTITY_WITH_D_T_O` instead of `CREATE_ENTITY_WITH_DTO`

### PR DESCRIPTION
**Pull request by cbornet**
Fix #27

### CODE REVIEW

File: reactive-grpc-common/src/main/java/com/salesforce/reactivegrpccommon/ReactiveGrpcGenerator.java [hunk 1]
```diff
@@ -172,7 +171,15 @@ private String absoluteFileName(ServiceContext ctx) {
         public int methodNumber;
 
         public String methodNameUpperUnderscore() {
```
Comment (rmichela) [inline reactive-grpc-common/src/main/java/com/salesforce/reactivegrpccommon/ReactiveGrpcGenerator.java]: Can you add a comment pointing to the gRPC-java code used to write this?

Comment (cbornet) [reply reactive-grpc-common/src/main/java/com/salesforce/reactivegrpccommon/ReactiveGrpcGenerator.java]: Yes, good idea.

**Comment by cbornet** [comment]
@rmichela if you have time to release a 0.7.2, I need it to progress on my jhipster-grpc module :bowing_man:

**Comment by rmichela** [comment]
Released.

**Comment by cbornet** [comment]
Awesome !

**Pull request merged**